File: Java8/src/InterestCalc.java
import java.util.Scanner;
/**
 * 
 *  @author admin
 *  @apiNote  
 *  This program is for interest calculation 
 */

public class InterestCalc {

	/** 
	 * it has 3 variable 
	 * amount, rate and time   
	 */
	public static void main(String[] args) {
		float amount;
		float rate;
		int time;

		Scanner scanner = new Scanner(System.in);

		do {
			System.out.println("enter amount");
			amount = scanner.nextFloat();
		} while (amount <= 0);

		do {
			System.out.println("enter rate");
			rate = scanner.nextFloat();
		} while (rate <= 0);

		do {
			System.out.println("enter time");
			time = scanner.nextInt();
		} while (time <= 0);

		float interest = amount * rate * time / 100;
		System.out.println(
				"amount is " + amount + "\nrate is " + rate + "\ntime is " + time + "\nInterest is " + interest);
		scanner.close();
	}
}

File: interest_calc.c
#include<stdio.h>
int main()
{
int time;
float amount,rate;

do 
{
	printf("\nenter amount");
   scanf("%f",&amount);
}
while(amount<=0);


do 
{
   printf("\nenter rate");
   scanf("%f",&rate);
}
while(rate<=0);

do 
{
   printf("\nenter time");
   scanf("%d",&time);
}
while(time<=0);

float interest=amount*rate*time/100;

printf("\ninterest  is %f ",interest);

return 0;

}

